client.js

```diff
@@ -15,31 +15,35 @@ let client;
 
 async function lookup({ _, ...rest }) {
   if (!rest.id) {
-    console.log("fetch required an ID");
+    console.log("lookup requires an ID");
     process.exit();
   }
-  console.log("Beginning client-side lookup: ", rest.id);
-  const lookedUpUser = await client.lookup({ id: rest.id });
-  console.log(lookedUpUser);
-  if (!lookedUpUser) {
-    console.log("Finished client-side lookup, found nothing");
-  } else {
-    console.log("Finished client-side lookup: ", lookedUpUser);
-  }
+
+  await client.lookup({ id: rest.id }, (err, user) => {
+    if (err) {
+      console.log("User not found");
```

Comment:
Maybe template out the User ID that wasn't found here?

Reply:
good point